### DISCUSSION TITLE
How to use queue for exporting large data

### DISCUSSION THREAD

**Discussion by MuhammadMesum514** [Power-Components/livewire-powergrid#449] [Q&A]
I want to export the records from database to excel using queues but I can't figure out how to do it. I have added the following code in my powergrid table file 
  
   `public int $queues = 2; // Use two queues
  
    public string $onQueue = 'ArReportInformation'; //queue name
  
    public string $onConnection = 'database'; // default sync
  
    public bool $showExporting = true; //Show progress on screen`

but when i click the export all button , it create a job and a job_batch record but there are not attempts. also there is nothing 
in failed jobs. can anybody show me how to do queue export. I have read documentation of livewire, powergrid and laravel but nothing seems to work

![Capture](https://user-images.githubusercontent.com/60551340/170562025-9ff82adf-7e25-4781-89dc-081ce4ffbebb.PNG)
![Capture2](https://user-images.githubusercontent.com/60551340/170562033-ed8cd4b9-2056-4e71-9255-c7f53f5c7b20.PNG)

**Comment by JhonArroyo** [comment]
try this `php artisan queue:work`

**Comment by marineusde** [comment]
Do you have the right configuration in your .env-File:

```
QUEUE_CONNECTION=database
```

instead of
```
QUEUE_CONNECTION=sync
```
